pkgs/tools/security/signify/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, libbsd, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "signify-${version}";
+  version = "v24";
+
+  src = fetchurl {
```

Comment:
Don't use fetchurl on GitHub source archives because these are regenerated from time to time and the SHA hash changes when this happens.
Instead you should use `fetchFromGitHub` or `fetchzip` (look at other packages for examples).

With both fetchers the new SHA hash will be computed over the whole content of the archive after unpacking.
You can compute it manually with `nix-prefetch-url --type sha256 --unpack https://github.com/aperezdc/signify/archive/v24.tar.gz`

Reply:
Good to know - thanks!